packages/calling-stateful-client/src/CallClientState.ts

```diff
@@ -101,6 +101,10 @@ export interface CaptionsCallFeatureState {
    * whether captions is on/off
    */
   isCaptionsFeatureActive: boolean;
+  /**
+   * whether start captions button is clicked or now
+   */
+  startCaptionsClicked: boolean;
```

Comment:
It could be a better name, it is not just representing the UI clicked, but also when someone call the function

Maybe like:
startCaptionsInProgress would work